app/assets/stylesheets/core/footer.scss

```diff
@@ -1,3 +1,4 @@
+// stylelint-disable no-descending-specificity
```

Comment:
Can we set this up in the config file instead of having to specify this per file?

Reply:
course! :)